Shared/Samples/Show device location using indoor positioning/ShowDeviceLocationUsingIndoorPositioningView.swift

```diff
@@ -0,0 +1,100 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import CoreLocation
+import SwiftUI
+
+struct ShowDeviceLocationUsingIndoorPositioningView: View {
+    /// The data model for the sample.
+    @StateObject private var model = Model()
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    /// This is the text description of the current floor that is displayed.
+    @State private var currentFloorText: String = ""
+    /// This is the text description of the data source that is displayed.
+    @State private var sourceText: String = ""
+    /// This is the text description of the number sensor of the data that is displayed.
+    @State private var sensorText: String = ""
+    /// This is the text description of the horizontal accuracy data that is displayed.
+    @State private var accuracyText: String = ""
+    
+    var body: some View {
+        MapView(map: model.map)
+            .locationDisplay(model.locationDisplay)
+            .overlay(alignment: .top) {
+                HStack(alignment: .center, spacing: 10) {
+                    VStack(alignment: .leading) {
+                        Text("Current floor: \(model.currentFloor ?? 0)")
+                        Text(
+                            """
+                          Accuracy: \(Measurement(value: model.horizontalAccuracy ?? 0.0, unit: UnitLength.meters), format: .measurement(width: .abbreviated, usage: .asProvided, numberFormatStyle: .number.precision(.fractionLength(2))))
+                          """
```

Comment:
A multiline string literal, as the name implies, should only be used when the string literal contains multiple lines.

```suggestion
                            "Accuracy: \(Measurement(value: model.horizontalAccuracy ?? 0.0, unit: UnitLength.meters), format: .measurement(width: .abbreviated, usage: .asProvided, numberFormatStyle: .number.precision(.fractionLength(2))))"
```

Reply:
Updated! Thanks for the tip!